os_stub/debuglib/debuglib.c

```diff
@@ -38,8 +38,10 @@ void libspdm_debug_assert(const char *file_name, size_t line_number, const char
 #if defined(_MSC_EXTENSIONS)
     __debugbreak();
 #endif
-#if defined(__GNUC__)
+#if defined(__GNUC__) && defined(__x86_64__)
```

Comment:
This interrupt is also available on `__i386__`?

Reply:
Oops, let me add that in.